Learner_Functions.py

```diff
@@ -0,0 +1,26 @@
+"""

```

Comment:
[PEP8 says](https://www.python.org/dev/peps/pep-0008/?#package-and-module-names) module names should be all lowercase.